src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -17,8 +17,7 @@ interface GetPathBreadcrumbsOpts {
 	/**
 	 * An array of nav nodes, as defined by our sidenav component.
 	 */
-
-	navData: NavData
+	navData: MenuItem[]
```

Comment:
It seems more correct to use our `MenuItem` interface here, plus it fixed some type errors from type updates in `docs-sidenav` 😄